src/DotNetClient/DotNetClient/Properties/AssemblyInfo.cs

```diff
@@ -20,16 +14,3 @@
 
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("14ef1cdd-1d99-4078-ab11-1207e859c63c")]
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version 
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.3.0")]
-[assembly: AssemblyFileVersion("1.0.3.0")]
```

Comment:
Has to be removed because dotnet standard source generates these based on  the csproj. So this causes duplicates.